### ISSUE TITLE
Allow app screenshots to be taken...

### ISSUE THREAD

**Issue by spandu500** [nic-delhi/AarogyaSetu_Android#27]
Don't see a reason why they're not allowed, can be useful in informing friends and family members about the stats in our area...

**Comment by ABcDexter** [comment]
What's the reason of not allowing screenshots :thinking:

**Comment by nihar1024** [comment]
I'm just guessing, but it's probably because Aarogya Setu is used as a "pass" app in multiple places (such as airports) to allow only "safe" citizens to access the space/service. Disallowing screenshots is a smart way to prevent people from showing a fake status.

**Comment by Electron-2002** [comment]
@nihar1024 There are many other ways to fake the screen. They can't disable taking screenshots just for that one reason is what I think. Maybe there is something else?

**Comment by spandu500** [comment]
> I'm just guessing, but it's probably because Aarogya Setu is used as a "pass" app in multiple places (such as airports) to allow only "safe" citizens to access the space/service. Disallowing screenshots is a smart way to prevent people from showing a fake status.

Well digilocker does allow app screenshots to be taken, what do you think of that? 🙈

**Comment by akx** [comment]
For what it's worth, the Helsinki metropolitan transit app in Finland has circumvented this by making the transit pass screen animated, with a color scheme that changes apparently daily, so screenshotting will not do. Of course that's not foolproof either.

**Comment by nihar1024** [comment]
@Electron-2002 I agree that there are other ways of faking your status, but doing it using screenshots is very easy for a large number of people. Disabling it means that it becomes a bit harder and requires more time and technical knowledge. Can't think of another reason they would disable it, apart from preventing misuse.
